music21/musicxml/m21ToXml.py

```diff
@@ -3358,6 +3405,7 @@ def parseFlatElements(
                     if hasSpannerAnchors:
                         self.parseOneElement(obj, AppendSpanners.NONE)
                     else:
+                        # ENTRY FOR Figured Bass Indications
```

Comment:
If you'd like to add a comment clarifying why harmony objects get handled here, I'd suggest improving the comment above ("including ChordSymbols or FiguredBassIndications not written as chord").
```suggestion
```